test/gzip.bats

```diff
@@ -0,0 +1,29 @@
+load helpers
+
+function setup() {
+    stacker_setup
+}
+
+function teardown() {
+    cleanup
+}
+
+@test "import tar" {
+  mkdir -p folder1
+  truncate -s 512k  folder1/file1
+  tar cvf test.tar folder1
+  cat > stacker.yaml <<EOF
+first:
+  from:
+    type: tar
+    url: test.tar
+EOF
+  stacker build
+  m1=$(cat oci/index.json | jq .manifests[0].digest | sed  's/sha256://' | tr -d \")
+  l1=$(cat oci/blobs/sha256/"$m1" | jq .layers[0].digest | sed  's/sha256://' | tr -d \")
+  skopeo copy --format=oci oci:oci:first containers-storage:test:tar
```

Comment:
Does the version of skopeo matter (since it also uses the image golibrary) ?

Reply:
It could, so just making sure we have a sane baseline.